### PR TITLE
Add Cockroach Community License with tests 

### LINKED ISSUE #858: Add Cockroach Community License (CCL)
See https://www.cockroachlabs.com/cockroachdb-community-license/
https://github.com/cockroachdb/cockroach/blob/master/LICENSE

### PR DESCRIPTION
**Pull request by nanspro**
Fixes #858 
Signed-off-by: Rahul Bishnoi rahulbishnoi3636@gmail.com

### CODE REVIEW

File: src/licensedcode/data/licenses/ccl.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+key: ccl
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/ccl.yml]: what about using a `cockroach` key instead? No offense meant to cockroaches, of course :)

File: src/licensedcode/data/licenses/ccl.LICENSE [hunk 2]
```diff
@@ -0,0 +1,127 @@
+CockroachDB Community License Agreement
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/ccl.LICENSE]: If possible it would be best if formatted on 80 chars per line

**Comment by nanspro** [comment]
@pombredanne  Cna you review this one more time

File: src/licensedcode/data/licenses/ccl.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+key: cockroach
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/ccl.yml]: You need to also rename the file for cockroach.yml and the text to cockroach.LICENSE

**Comment by pombredanne** [comment]
Did you run the tests locally? And did you check the Travis/Appveyor tests results?

**Comment by pombredanne** [comment]
Ping?

**Comment by yash-nisar** [comment]
@pombredanne Is this being worked upon ? If no, can I be assigned to this one ?

**Comment by pombredanne** [comment]
@yash-nisar this being a PR, this is being worked on alright, unless @nanspro wants to pass the baton?

**Comment by yash-nisar** [comment]
Okay @pombredanne , lets wait for a couple of days. If the assignee doesn't respond, I'll complete the patch. How does this sound ?

**Comment by pombredanne** [comment]
@nanspro thanks you for all your efforts but since you were not reposnding I went with @yash-nisar PR instead #891 as he followed through to the end.

**Pull request closed without merging**